docs/source/start.rst

```diff
@@ -239,12 +239,12 @@ Name: sepal_length, dtype: float64
 
 This selects all even rows in the ``sepal_length`` column:
 
->>> idadf_new = idadf.loc[::2,'sepal_length']
+>>> idadf_new = idadf.loc[::2][['ID', 'sepal_length']]
```

Comment:
Reading the file from top to bottom, it's not clear where the `ID` column is coming from. It did not exist in the earlier access of `idadf.columns`.

Reply:
Yes I agree, the comment just above this line of of code assumes it already exists. I will add a code step to make this clearer.